src/Microsoft.Diagnostics.Monitoring.EventPipe/Configuration/HttpRequestSourceConfiguration.cs

```diff
@@ -16,7 +16,7 @@ public HttpRequestSourceConfiguration()
             RequestRundown = true;
         }
 
-        private const string DiagnosticFilterString =
+        internal const string DiagnosticFilterString =
```

Comment:
Can you add some comments around the reasoning for single sourcing this across multiple configs

Reply:
I added a comment for this - let me know if you'd prefer different wording